documentation/library_guide/parallel_api/parallel_range_algorithms.rst

```diff
@@ -106,6 +106,13 @@ Usage Example for Parallel Range Algorithms
         oneapi::dpl::ranges::copy(oneapi::dpl::execution::dpcpp_default, view_in, std::span(vec_out));
     }
 
+Potential Issues
+----------------
+The ``sort`` and ``stable_sort`` algorithms provided use ``std::swap`` internally instead of ``std::ranges::iter_swap``
+for swapping elements. Note that the ``std::ranges::sort`` algorithm in several C++ standard library
+implementations also does not make use of ``std::ranges::iter_swap``. The potential issue is that customizations
+targeting ``std::ranges::iter_swap`` will not be respected.
+
```

Comment:
I suggest slightly rephrasing the note and changing the title. 

```suggestion
Implementation Notes
---------------------------
 - The sort and ``stable_sort`` algorithms in the ``oneapi::dpl::ranges`` namespace use ``std::swap`` internally instead of ``std::ranges::iter_swap`` for swapping elements, as the C++ standard suggests but does not explicitly require.
 As a result, customizations targeting ``std::ranges::iter_swap`` will not be respected.

```

We can also add this note together with other related changes.

Signatures of some algorithms do not conform C++ standard and it is expected to be mentioned later (as noted in https://github.com/oneapi-src/oneDPL/pull/1596#discussion_r1758917326). There is a similarity with this issue, even though the addressed discrepancy is more subtle. They are likely to fit the same section, which will be more complete.